middleware/kubernetes/kubernetes_test.go

```diff
@@ -212,36 +194,6 @@ func TestParseRequest(t *testing.T) {
 	for field, expected := range tcs {
 		expectString(t, f, "TXT", query, &r, field, expected)
 	}
-
-	// Invalid query tests
```

Comment:
Perhaps changes these tests to validate NODATA response, instead of just removing them.